### PR TITLE
Improve codes for showing run_setup

### PR DESCRIPTION
**Pull request by chuan-wang**

### CODE REVIEW

File: ngi_reports/reports/project_summary.py [hunk 1]
```diff
@@ -229,7 +229,29 @@ def __init__(self, config, LOG, working_dir, **kwargs):
...
-            run_setup = fc_obj.get("run_setup")
+            ## Fetch run setup for the flowcell
+            run_setup = fc_obj.get("RunInfo").get('Reads')
+            ## Sort by the order of reads
+            run_setup = sorted(run_setup, key=lambda k: k['Number'])
+            run_setup_text = ''
+            read_count = 0
+            index_count = 0
+            for read in run_setup:
+                run_setup_text += read['NumCycles'].encode("utf-8")
```
Comment (aanil) [inline ngi_reports/reports/project_summary.py:240]: We don't need to encode it to utf-8, since we'll run it on python3

Comment (chuan-wang) [reply ngi_reports/reports/project_summary.py]: Thx! Fixed! /chuan

**Review by ssjunnebo — APPROVED** [review]
Looks good! 👍

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**